spec/cpp_library_spec.rb

```diff
@@ -55,6 +55,18 @@ def get_relative_dir(sampleprojects_tests_dir)
   context "test" do
     config = ArduinoCI::CIConfig.default
 
+    puts "\n\nstring.h\n\n"
+    puts File.read("c:/cygwin/usr/include/string.h")
+
+    puts "\n\n_ansi.h\n\n"
+    puts File.read("c:/cygwin/usr/include/_ansi.h")
+
+    puts "\n\nsys/config.h.h\n\n"
+    puts File.read("c:/cygwin/usr/include/sys/config.h")
+
+    puts "\n\nsys/features.h.h\n\n"
+    puts File.read("c:/cygwin/usr/include/sys/features.h")
+
```

Comment:
I assume this is for debugging and not part of the final PR

Reply:
Yup, everything in the last commit was for debugging.